wicket-user-guide/pom.xml

```diff
@@ -32,7 +32,7 @@
 
 	<properties>
 		<automatic-module-name>org.apache.wicket.userguide</automatic-module-name>
-		<asciidoctor.version>1.5.5</asciidoctor.version>
+		<asciidoctor.version>2.2.2</asciidoctor.version>
```

Comment:
Please doublecheck that the user guide is properly generated, both HTML and PDF

Reply:
Thanks for the review @martin-g,
Guide build now works for me
But, I guess, the build need to be tested on Windows.

Do you have Windows machine to test? :)

Reply:
No, sorry!
I have the feeling that we will need to revert the bump of the asciidoctor changes but let's wait some time for feedback from a Windows user.

Reply:
sure, let's wait :))
we can drop these changes for 9.x and move them to master :)))

Reply:
No complains from Windows users!
I think we can merge the PR!